Jellyfin.Plugin.TubeArchivistMetadata/Plugin.cs

```diff
@@ -66,15 +66,15 @@ public Plugin(
 
             var taToJellyfinProgressSyncTask = new TAToJellyfinProgressSyncTask(logger, libraryManager, userManager, userDataManager);
             var jfToTubearchivistProgressSyncTask = new JFToTubearchivistProgressSyncTask(logger, libraryManager, userManager, userDataManager);
-            var isTAJFTaskPresent = taskManager.ScheduledTasks.Any(t => t.Name.Equals(taToJellyfinProgressSyncTask.Name, StringComparison.Ordinal));
+            var isTAJFTaskPresent = taskManager.ScheduledTasks.Any(t => t.ScheduledTask.Name.Equals(taToJellyfinProgressSyncTask.Name, StringComparison.Ordinal));
             if (Instance!.Configuration.TAJFSync && !isTAJFTaskPresent)
             {
                 logger.LogInformation("Queueing task {TaskName}.", taToJellyfinProgressSyncTask.Name);
                 taskManager.AddTasks([taToJellyfinProgressSyncTask]);
                 taskManager.Execute<TAToJellyfinProgressSyncTask>();
             }
 
-            var isJFTATaskPresent = taskManager.ScheduledTasks.Any(t => t.Name.Equals(jfToTubearchivistProgressSyncTask.Name, StringComparison.Ordinal));
+            var isJFTATaskPresent = taskManager.ScheduledTasks.Any(t => t.ScheduledTask.Name.Equals(jfToTubearchivistProgressSyncTask.Name, StringComparison.Ordinal));
```

Comment:
The type for `ScheduledTasks` appears to have changed in the upstream `ITaskManager` so this just makes sure the names are compared as before